flake.nix

```diff
@@ -51,13 +51,32 @@
             cargo = rust;
             rustc = rust;
           }).buildRustPackage;
+
+          # Filter out VCS files and files unrelated to the Rust ragenix package
+          filterRustSource = src: with lib; cleanSourceWith {
+            filter = cleanSourceFilter;
+            src = cleanSourceWith {
+              inherit src;
+              filter = name: type:
+                let pathWithoutPrefix = removePrefix (toString src) name; in
+                  ! (
+                    hasPrefix "/.github" pathWithoutPrefix ||
+                    pathWithoutPrefix == "/.gitignore" ||
+                    pathWithoutPrefix == "/LICENSE" ||
+                    pathWithoutPrefix == "/README.md" ||
+                    pathWithoutPrefix == "/flake.lock" ||
+                    pathWithoutPrefix == "/flake.nix"
```

Comment:
Maybe it is better to work with suffixes here, like filtering `.nix` and `.md` files, so that we do not have to touch this here too often.

Reply:
Nix files are also part of the rust code (tests).

In general, I think it is a good idea to keep this list explicit instead of excessive globbing.

Reply:
Ah ok, tehn it's fine